spec/21-http_response_header_format.md

```diff
@@ -15,9 +15,9 @@ The `traceresponse` HTTP response header field identifies a completed request in
 
 Header name: `traceresponse`
 
-In order to increase interoperability across multiple protocols and encourage successful integration, the header name SHOULD be lowercase. The header name is a single word without any delimiters, for example, a hyphen (`-`).
+The header name is [ASCII case-insensitive](https://infra.spec.whatwg.org/#ascii-case-insensitive). That is, `TRACERESPONSE`, `TraceResponse`, and `traceresponse` are considered the same header. The header name is a single word, it does not contain any delimiters such as a hyphen.
```

Comment:
> The header name is a single word, it does not contain any delimiters such as a hyphen

This is a comma splice

```suggestion
The header name is [ASCII case-insensitive](https://infra.spec.whatwg.org/#ascii-case-insensitive). That is, `TRACERESPONSE`, `TraceResponse`, and `traceresponse` are considered the same header. The header name is a single word; it does not contain any delimiters such as a hyphen.
```

source: https://owl.purdue.edu/owl/general_writing/punctuation/commas/commas_vs_semicolons.html